Author: Charles Plessy
Forwarded: no
 This patch uses a specificity of Debian.
Description: Browse the documentation with default pager.
 `/usr/bin/pager', always exists in Debian and is managed through the
 alternatives system. With this patch, `tfm' will use `/usr/bin/pager' by
 default, which points on the user's favorite pager.
License: as if it were public domain.
--- emboss.orig/emboss/tfm.c
+++ emboss/emboss/tfm.c
@@ -87,7 +87,7 @@
 	    if(shellpager)
 		ajStrAssignC(&pager,shellpager);
 	    if(!ajStrGetLen(pager))
-		ajStrAssignC(&pager,"more");
+		ajStrAssignC(&pager,"pager");
 	}
 	ajFmtPrintS(&cmd,"%S %S",pager,path);
 	ajSysExecPathS(cmd);
